stingray/tests/test_lightcurve.py

```diff
@@ -614,3 +614,8 @@ def test_lc_baseline(self):
         lc = Lightcurve(times, counts, gti=gti)
         baseline = lc.baseline(10000, 0.01)
         assert np.all(lc.counts - baseline < 1)
+
```

Comment:
Changes in `test_lightcurve` were made because coverage tests were failing although `test_crosscorrelation` had full coverage. This, ideally should have been separate PR but since it modified just a 3 to 4 lines, I added it as a commit in this PR. This has nothing to do with `CrossCorrelation`. Just to ensure that coverage tests pass.